css/main.css

```diff
@@ -1,11 +1,6 @@
-body {
-	padding-top: 3em;
-	max-width: 45em;
-	margin: auto;
-	text-align: center;
-	font-family: 'Helvetica Neue', Arial, Helvetica, sans-serif;
+.SGMAPembauche {
 	font-size: 150%;
-	line-height: 2.5;
```

Comment:
Same thing here, I'd consider `line-height` to be part of our text-based interface. It is still overridable by clients, but I'm afraid the widget looks bloated without it.